MVP/defaults-build.yaml

```diff
@@ -26,6 +26,8 @@
         - timestamps
         - build-name:
             name: ${{ENV,var="name"}}-${{ENV,var="version"}}-${{ENV,var="release"}}
+        - inject:
+            script-content: echo "create_time=$(date --utc +%FT%TZ)" >> $WORKSPACE/job.properties
```

Comment:
I think this happens before shell build step, right?
Will it be overwritten while injecting BKR_ID later (e.g. sample_job.yaml line 31) ?

Thus we'd use ">>" there (however we do workspace cleanup every time so it's fine to use either ">" or ">>" here).